Sources/TSCLibc/libc.swift

```diff
@@ -18,3 +18,12 @@
 #endif
 
 @_exported import TSCclibc
+
+#if os(Windows)
+// char *realpath(const char *path, char *resolved_path);
+public func realpath(_ path: String,
```

Comment:
Nit-pick: can you format the function closer to what we generally do in SwiftPM?

```swift
public func realpath(
    _ path: String,
    _ resolvedPath: UnsafeMiutableBufferPointer<CChar>?
) -> UnsafePointer<CChar>? {
   fatalError("realpath is unimplemented")
 }
```